app/components/auth/StepsContainer.js

```diff
@@ -45,14 +57,32 @@ type Props = {
   navigateToWallet: Function
 };
 
-class StepsContainer extends PureComponent<Props> {
+type State = {
+  redirectToLocalNode: boolean,
+  allowClose: boolean
+};
+
+class StepsContainer extends PureComponent<Props, State> {
+  state = {
+    redirectToLocalNode: false,
+    allowClose: true
+  };
+
   render() {
     const { mode } = this.props;
+    const { redirectToLocalNode, allowClose } = this.state;
     const header = mode === authModes.UNLOCK ? 'Welcome Back' : 'Welcome to Spacemesh';
+    if (redirectToLocalNode) {
```

Comment:
this is not supposed be to here, redirection decision should be made somewhere else and sure not in render func